game/tests/test_services.py

```
@@ -0,0 +1,243 @@
+
+        # openings_completed should increment only the first time an opening is completed
+        progress, first = update_opening_progress(self.user, "Italian Game", completed=True)
+        self.assertTrue(first)
+        self.assertEqual(progress.openings_completed, 1)
+
+        progress, first = update_opening_progress(self.user, "Italian Game", completed=True)
+        self.assertFalse(first)
+        self.assertEqual(progress.openings_completed, 1)
+
+        # Newly created progress should initialize openings_started = 1
+        progress2, first2 = update_opening_progress(self.user, "Ruy Lopez")
```

Ruff (0.15.18)

[warning] Unpacked variable `first2` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)